merlin/router.py

```diff
@@ -103,12 +104,13 @@ def purge_tasks(task_server, spec, force, steps):
     """
     LOG.info(f"Purging queues for steps = {steps}")
 
-    if task_server == "celery":
+    if task_server == "celery":  # pylint: disable=R1705
         queues = spec.make_queue_string(steps)
         # Purge tasks
         return purge_celery_tasks(queues, force)
     else:
         LOG.error("Celery is not specified as the task server!")
+        return -1
```

Comment:
Why the -1? Is this consistent with what we do elsewhere? Should we pin a to-do if creating an enum of return codes?

Reply:
I originally added this because pylint was complaining that there should be a return from every path in the function. I chose -1 for now as the return code to represent that something went wrong. This function in particular only returns a value for logging purposes so we could change this to not return anything and just log inside router.py rather than main.py.

Right now the return values from every function in router.py are not consistent (i.e. some return ints, some return lists, some don't return anything, etc.). The enum of return codes isn't a bad idea but it would require a slightly larger to-do as we'd have to refactor the return values of a lot of these functions. I'm all for creating a to-do for this, just warning that it might take more work than we think.